crates/ef-testing/src/evm_sequencer/mod.rs

```diff
@@ -86,15 +86,16 @@ impl KakarotSequencer {
                     .map_err(|err| StateError::ProgramError(ProgramError::Parse(err)))?,
             )?),
         )?;
-        (&mut self.0.state).set_class_hash_at(*FEE_TOKEN_ADDRESS, *FEE_TOKEN_CLASS_HASH)?;
+        (&mut self.0.state).set_class_hash_at(*ETH_FEE_TOKEN_ADDRESS, *FEE_TOKEN_CLASS_HASH)?;
 
         Ok(self)
     }
 }
 
 impl Execution for KakarotSequencer {
     fn execute(&mut self, transaction: Transaction) -> Result<(), TransactionExecutionError> {
-        self.0.execute(transaction)
+        let _ = self.0.execute(transaction);
```

Comment:
why do we need this?

Reply:
I used to return the error, which caused the runner to crash on any transaction execution or validation issue. But the way the ef-tests are designed is for any failing transaction to be caught via the post state. So if your tx is supposed to fail, this will be reflected in the post state. An example: the tx I send to the contract will attempt to write 1 in the storage. However, this tx should fail because of XYZ. The post state should then NOT contain 1 in the storage.